Sources/IO/Geometry/STLReader/index.js

```diff
@@ -331,6 +331,60 @@ function vtkSTLReader(publicAPI, model) {
   publicAPI.requestData = (inData, outData) => {
     publicAPI.parse(model.parseData);
   };
+
+  publicAPI.removeDuplicateVertices = (offset = 5) => {
+    if (!model.output || !model.output[0]) {
+      console.warn('Load polydata first.');
+      return;
+    }
+    const polydata = model.output[0];
+
+    const vertices = polydata.getPoints().getData();
```

Comment:
a `vertex` is a "cell" in the VTK world, it's better to call it `points` or `coords`